src/encoded/tests/test_types_donor.py

```diff
@@ -0,0 +1,29 @@
+import pytest
+
+
+@pytest.fixture
+def parent(testapp, award, lab, human):
+    item = {
+        'lab': lab['@id'],
+        'award': award['@id'],
+        'organism': human['@id']
+    }
+    return testapp.post_json('/human_donor', item).json['@graph'][0]
+
+
+@pytest.fixture
+def child(testapp, award, lab, human):
+    item = {
+        'lab': lab['@id'],
+        'award': award['@id'],
+        'organism': human['@id']
+    }
+    return testapp.post_json('/human_donor', item).json['@graph'][0]
+
+
+# patch parent to child, check that children are a calculated property for the parent
+def test_types_calculated_children(testapp, parent, child):
+    testapp.patch_json(child['@id'],
+                             {'parents': [parent['@id']]}).json['@graph'][0]
+    res = testapp.get(parent['@id'] + '@@index-data')
+    assert res.json['object']['children'] == [child['@id']]
```

Comment:
Doesn't `res.json['children']` work?

Reply:
It seems like it should, but removing ['object'] results in KeyError: 'children'.

Other tests also use the res.json['object']['property'] form.

Reply:
Oh, yeah, I see. It is because you are getting the `'@@index-data'` endpoint.